scripts/imgtool/main.py

```diff
@@ -235,6 +235,8 @@ def convert(self, value, param, ctx):
                    'to 128)')
 @click.option('--pad', default=False, is_flag=True,
               help='Pad image to --slot-size bytes, adding trailer magic')
+@click.option('--confirm', default=False, is_flag=True,
```

Comment:
Please just move this new option above `--pad` so it shows afterwards in the help page.

Reply:
Fixed.